src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,56 @@
+package seedu.addressbook.common;
+
+import java.util.*;
+import seedu.addressbook.common.Utils;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test; 
+
+public class UtilsTest {
+	
+	@Before public void setUp() {
+		private final firstObject = "";
+		private final secondObject = "";
+		private final thirdObject = "";
+		private final forthObject = " ";
+		private final fifthObject = "1";
```

Comment:
- These variables are missing their type.
- It might have been more meaningful to ensure that these differently named variables have different values.